site/en/docs/web-platform/launch-handler/index.md

```diff
@@ -16,6 +16,10 @@ The Launch Handler API lets you control how your app is launched, for example, w
 existing or a new window and whether the chosen window is navigated to the launch URL. As with
 the File Handing API, this also enqueues a `LaunchParams` object in the launched page's `window.launchQueue`.
 
+{% Aside 'important' %}
+Launch handling and link capturing are often confused, but they are actually two different things. Link capturing is the decision of the operating system on how in-scope links open, either in a browser tab or in an installed PWA. [Declarative link capturing](/docs/web-platform/declarative-link-capturing/) was a proposed API to let developers control that behavior but it will not launch, so it's currently up to the operating system to decide how to implement this.
```

Comment:
I would phrase this sentence on DLC slightly differently:

1. "Declarative link capturing was a proposed API to control both of these aspects, but it was removed in favour of Launch Handling, leaving the details of link capturing up to the operating system and user configuration."

Reply:
@demianrenzulli, did you also mean to mention the `handle_links` manifest member?  https://github.com/WICG/pwa-url-handler/blob/main/handle_links/explainer.md

Reply:
> I would phrase this sentence on DLC slightly differently:
> 
> "Declarative link capturing was a proposed API to control both of these aspects, but it was removed in favour of Launch Handling, leaving the details of link capturing up to the operating system and user configuration."

@mgiuca, to avoid even more misunderstandings, I would refrain from writing "both of these aspects" in favor of calling them out explicitly by name, especially given that an aside ideally should be self-contained and make sense without reading the surrounding text.